doc/source/ray-observability/ray-logging.rst

```diff
@@ -166,6 +166,7 @@ Here's a Ray log directory structure. Note that ``.out`` is logs from stdout/std
 - ``raylet.[out|err]``: A log file of raylets.
 - ``redis-shard_[shard_index].[out|err]``: Redis shard log files.
 - ``redis.[out|err]``: Redis log files.
+- ``runtime_env_agent.log``: Every Ray node has one agent that manages :ref:`runtime environment <runtime-environments>` creation, deletion and caching. This is the log file of the agent.
```

Comment:
Just wondering - how does `creation` part differ from the `installing` part mentioned for `runtime_env_setup-` log below? Maybe could modify this description if this could become a possible misunderstanding

Reply:
Yup this is totally unclear, I'll update it

Reply:
`runtime_env_agent.log` would have the logs from the raylet sending the create request and delete requests to the agent, and cache hits/misses, but the `setup-<job-id>` logs would have the actual installation logs per job from the `pip install`, `conda install` and so on.

Reply:
Got it, that makes more sense, yeah feel free to add that to the description